ReactAndroid/src/main/java/com/facebook/react/modules/core/DeviceEventManagerModule.java

```diff
@@ -53,6 +56,15 @@ public void emitHardwareBackPressed() {
   }
 
   /**
+   * Sends an event to the JS instance that a new intent was received.
+   */
+  public void emitNewIntent(Uri uri) {
```

Comment:
The docs are good!

Should this be called `emitNewIntentReceived` for clarity? By looking at the name itself I can't tell whether this creates a new intent somehow or whether it notifies JS the intent was received.